internal/services/monitor/monitor_private_link_scope_resource.go

```diff
@@ -47,6 +48,30 @@ func resourceMonitorPrivateLinkScope() *pluginsdk.Resource {
 				ValidateFunc: validate.PrivateLinkScopeName,
 			},
 
+			"ingestion_access_mode": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(privatelinkscopesapis.AccessModeOpen),
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(privatelinkscopesapis.AccessModeOpen),
+						string(privatelinkscopesapis.AccessModePrivateOnly),
+					},
+					false),
```

Comment:
We can replace this with 
```suggestion
				ValidateFunc: validation.StringInSlice(privatelinkscopesapis.PossibleValuesForAccessMode(), false),
```